tests/unit/fem/test_pa_coeff.cpp

```diff
@@ -57,7 +57,7 @@ TEST_CASE("H1 pa_coeff")
             std::cout << "Testing " << dimension << "D partial assembly with "
                       << "coeffType " << coeffType << " and "
                       << "integrator " << integrator << std::endl;
-            for (int order = 1; order < 5; ++order)
+            for (int order = 1; order < 4; ++order)
             {
```

Comment:
@dylan-copeland, any objection to reduce the order in this test?

It speeds-up the unit testing significantly. Compare for example these Travis results ([old](https://travis-ci.org/mfem/mfem/builds/659637428) vs [new](https://travis-ci.org/mfem/mfem/builds/659691060)):

- **Linux Serial:** [old, 19.24s](https://travis-ci.org/mfem/mfem/jobs/659637433#L694) vs [new, 6.05s](https://travis-ci.org/mfem/mfem/jobs/659691065#L695)
- **Linux Parallel:** [old, 20.81s](https://travis-ci.org/mfem/mfem/jobs/659637435#L989) vs [new, 7.00s](https://travis-ci.org/mfem/mfem/jobs/659691067#L991)
- **Mac Serial:** [old, 25.02s](https://travis-ci.org/mfem/mfem/jobs/659637437#L523) vs [new, 8.27s](https://travis-ci.org/mfem/mfem/jobs/659691069#L524)
- **Mac Parallel:** [old, 66.77s](https://travis-ci.org/mfem/mfem/jobs/659637439#L664) vs [new 18.81s](https://travis-ci.org/mfem/mfem/jobs/659691071#L662)

Reply:
@camierjs, in the above runs, do we expect the time for `mpirun -np 4 sedov_tests_debug` in parallel and `sedov_tests_debug` in serial to be relatively high?

Reply:
Yes, I have looked at this issue in another branch to speed it up, especially on MacOS.

Reply:
@tzanio No objections. If something breaks the lower orders should hopefully catch it.